src/source.cpp

```diff
@@ -0,0 +1,78 @@
+// Source.cpp
+// Central Source Class
+
+#include <iomanip>
+#include <string>
+#include <vector>
+
+#ifndef PYNE_IS_AMALGAMATED
+#include "particle.h"
+#include "source.h"
+#endif
+
+/***************************/
+/*** Protected Functions ***/
+/***************************/
+
+// there are no protected functions currently
+// fool.
+
+/************************/
+/*** Public Functions ***/
+/************************/
+
+/*--- Constructors ---*/
+
+pyne::Source::Source() {}
+
+// Destructor
+pyne::Source::~Source() {}
+
+pyne::PointSource::PointSource(double _x, double _y, double _z, double _u, double _v,
+                               double _w, double _E, std::string _particle, double _weight)
+    : x(_x), y(_y), z(_z), u(_u), v(_v), w(_w), E(_E), particle(_particle), weight(_weight) {}
+
+// Destructor
+pyne::PointSource::~PointSource() {}
+
+std::string pyne::PointSource::mcnp(int version) {
+  std::stringstream output;  // output stream
+  
+  std::stringstream newline;
+  newline << std::endl << "     ";
+  
+  std::cout << __LINE__ << std::endl; 
```

Comment:
I don't understand what `__LINE__` is?

Do we really want to write to `std::cout` always?

Reply:
sorry forgot tho clean that up :(